src/constants.ts

```diff
@@ -105,3 +105,54 @@ export const SHORTLINK_HOST_BY_PAGE_HOST = {
   'test.viewer.swissgeol.ch': 'link.swissgeol.ch',
   'viewer.swissgeol.ch': 'link.swissgeol.ch',
 };
+
+export const SHOWCASE_PROJECTS = [
```

Comment:
I thinks this should be put in a static file and fetched when the dashboard is opened for the first time.

Reply:
updated